### ISSUE TITLE
Support for babel 7

### ISSUE THREAD

**Issue by aprisniak** [ctumolosus/jsdoc-babel#82]
Did you have plans for adding support for babel 7?

**Comment by ctumolosus** [comment]
Just published `jsdoc-babel@0.4.0-alpha.0` with support for Babel 7 (beta) - It's intentionally a pre-release until Babel 7 becomes stable. You can use the provided [example](https://github.com/ctumolosus/jsdoc-babel/tree/next/example) to guide you, it shouldn't be that much different from the previous version.

By the way, JSDoc has been using Babylon to parse JavaScript files as of version 3.5.0, you may not need this plugin unless you need to transform unsupported syntax.

**Comment by aprisniak** [comment]
@ctumolosus Thx, it works! :) And I need `jsdoc-babel` because jsdoc doesn't work with flow (

**Comment by bensampaio** [comment]
@ctumolosus this is not working for me. I am also using flow with `babel@7.0.0-beta.51` and `jsdoc-babel@0.4.0-alpha.0`. I have a `babel.config.js` for babel configuration and I added the plugin to the jsdoc config but I still get the following errors:

```
ERROR: Unable to parse /Users/bensampaio/Repositories/karify/frontends/lib/react/src/components/Avatar/Avatar.jsx: Unexpected token (5:12)
ERROR: Unable to parse /Users/bensampaio/Repositories/karify/frontends/lib/react/src/components/Icon/Icon.jsx: Unexpected token (5:12)
```

**Comment by ctumolosus** [comment]
It's hard for me to picture your setup. The one thing that comes to mind is that Babel looks for a `.babelrc` file in your project and you have `babel.config.js`. I also don't think Babel has an option to specify a different config file.

**Comment by bensampaio** [comment]
In babel 7 ʼ.babelrcʼ was recently renamed to ʼbabel.config.jsʼ. Maybe that's the issue with this plugin?

**Comment by ctumolosus** [comment]
I don't think so. This plugin does not try to resolve the configuration file on behalf of Babel. Is there a link to a repository that you can share where I can reproduce the error? Or, can you put something together on [code sandbox](https://codesandbox.io/) or similar?

**Comment by ctumolosus** [comment]
Official support released on version 0.5.0 (#91)
